subsys/net/lib/lwm2m/lwm2m_message_handling.c

```diff
@@ -3131,3 +3111,13 @@ int lwm2m_engine_send(struct lwm2m_ctx *ctx, char const *path_list[], uint8_t pa
 	return -ENOTSUP;
 #endif
 }
+
+__deprecated
```

Comment:
```suggestion
```
I think this deprecation in C file is unnecessary, just move it into the header.

Reply:
Done